wallet_createwallet.py --legacy-wallet fails on a system with only bdb installed

```
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/bitcoin-core/test/functional/test_framework/test_framework.py", line 133, in main
    self.run_test()
  File "/tmp/cirrus-ci-build/bitcoin-core/test/functional/wallet_createwallet.py", line 31, in run_test
    assert_raises_rpc_error(-4, "Passphrase provided but private keys are disabled. A passphrase is only used to encrypt private keys, so cannot be used for wallets with private keys disabled.",
  File "/tmp/cirrus-ci-build/bitcoin-core/test/functional/test_framework/util.py", line 125, in assert_raises_rpc_error
    assert try_rpc(code, message, fun, *args, **kwds), "No exception raised"
  File "/tmp/cirrus-ci-build/bitcoin-core/test/functional/test_framework/util.py", line 140, in try_rpc
    raise AssertionError(
AssertionError: Expected substring not found in error message:
substring: 'Passphrase provided but private keys are disabled. A passphrase is only used to encrypt private keys, so cannot be used for wallets with private keys disabled.'
error message: 'Compiled without sqlite support (required for descriptor wallets)'.

--- a/test/functional/wallet_createwallet.py
+++ b/test/functional/wallet_createwallet.py
@@ -29,7 +29,7 @@ class CreateWalletTest(BitcoinTestFramework):
         self.log.info("Run createwallet with invalid parameters.")
         # Run createwallet with invalid parameters. This must not prevent a new wallet with the same name from being created with the correct parameters.
         assert_raises_rpc_error(-4, "Passphrase provided but private keys are disabled. A passphrase is only used to encrypt private keys, so cannot be used for wallets with private keys disabled.",
-            self.nodes[0].createwallet, wallet_name='w0',  descriptors=True, disable_private_keys=True, passphrase="passphrase")
+            self.nodes[0].createwallet, wallet_name='w0', disable_private_keys=True, passphrase="passphrase")
 
         self.nodes[0].createwallet(wallet_name='w0')
         w0 = node.get_wallet_rpc('w0')

